homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -179,11 +176,23 @@ def device_class(self) -> str | None:
         """Return the class of this device, from component DEVICE_CLASSES."""
         return self._sensor_data.get(ATTR_DEVICE_CLASS)
 
+    @property
+    def state_class(self) -> str | None:
+        """Return the class of the state of this device, from component STATE_CLASSES."""
+        return self._sensor_data.get(ATTR_STATE_CLASS)
+
     @property
     def entity_registry_enabled_default(self) -> bool:
         """Return if the entity should be enabled when first added to the entity registry."""
         return self._sensor_data.get(ATTR_ENABLED_DEFAULT, False)
 
+    @property
+    def last_reset(self) -> datetime | None:
+        """Display a generic last_reset time when ATTR_STATE_CLASS is set"""
+        if ATTR_STATE_CLASS in self._sensor_data:
+            return utc_from_timestamp(0)
```

Comment:
last reset should only be used for metered entities. And if it's static, why not add it to your sensor description object?

Reply:
Just address this, and we can merge.

Reply:
Just pushed the requested changes.